style.css

```
@@ -40,3 +43,56 @@ p {
   box-shadow: 0 0 25px rgba(0, 0, 0, 0.25);
   background: white;
 }
+
+.list{
+    margin:0px;
+    padding-left:0px;
+}
+
+ul{
+  padding-left: 15px;
+}
+
+.js-doneList{
+  text-decoration: line-through; /*왜 적용이 안되는지*/
+  color: gray;
+}
```

```suggestion
.js-doneList{
  color: gray;
}

.js-doneList > li{
  text-decoration: line-through;
}
```
`text-decoration` 속성은 상속이 안되기 때문에 `ul`인 `.js-doneList`에 적용하기 보다는 `ul` 안에있는 `li`에 적용해야 합니다!

---

적용 안된 이유가 여기 있었군요...감사합니다